Source/WebCore/Modules/compression/DecompressionStreamDecoder.h

```diff
@@ -64,14 +74,35 @@ class DecompressionStreamDecoder : public RefCounted<DecompressionStreamDecoder>
     bool finish { false };
     z_stream zstream;
 
+    bool usingAppleCompressionFramework { false };
```

Comment:
All member variables should start with m_, including stream, finish, stream...
Can you update DecompressionStreamDecoder (and Encoder if needed), in a  separate PR probably.

Reply:
Thanks will fix

Reply:
Should function pointers also have `m_` in front?

`ExceptionOr<Vector<uint8_t>> (DecompressionStreamDecoder::*decompress)(const uint8_t*, const size_t);`

Reply:
I would guess so